src/commands/dev/list.ts

```diff
@@ -59,11 +59,7 @@ export default class DevList extends BaseCommand {
         };
       }
     }
-    if (!Object.keys(output).length) {
```

Comment:
Talked offline, but keep the friendly error msg unless they use format json

Reply:
that's what this is doing I believe - this is within the `outputJSON` function and the other table output still has the error msg

Reply:
This does do that. The output table function still produces the message. Only the output json function had this removed. Unless you mean something else.

Reply:
Tested changes for sanity check. 

```
architect-local dev:list --format JSON
{}
architect-local dev:list --format json
{}
architect-local dev:list --format table
There are no active dev instances yet. Use `architect dev` to create one.
architect-local dev:list --format TABLE
There are no active dev instances yet. Use `architect dev` to create one.
architect-local dev:list
There are no active dev instances yet. Use `architect dev` to create one.
```

Reply:
This is my fault for the horrific naming of this PR. I've updated the title to be more specific and description to include testing results.

Reply:
ah my bad - assumed there was one output flow